client/my-sites/plugins/main.jsx

```diff
@@ -440,6 +434,9 @@ export default connect(
 			canJetpackSiteUpdateFiles: siteId => canJetpackSiteUpdateFiles( state, siteId ),
 			isJetpackSite: siteId => isJetpackSite( state, siteId ),
 			wporgPlugins: state.plugins.wporg.items,
+			userCanManagePlugins: ( !! selectedSite
+				? canCurrentUser( state, selectedSite.ID, 'manage_options' )
```

Comment:
> Anyway, we don't have siteId variable that we can use do you think we should create one using getSelectedSiteId ?

Yes, this is common practice, and it's generally safer to fetch particular attributes with dedicated selectors, rather than manually doing what the selector would do. This approach is also sustainable in case we ever decide to change the state tree structure.